src/annotator/integrations/test/pdf-test.js

```diff
@@ -312,7 +312,7 @@ describe('PDFIntegration', () => {
 
       assert.isTrue(active);
       assert.calledOnce(fakePDFViewerApplication.pdfViewer.update);
-      assert.equal(pdfContainer().style.width, 1350 - 428 + 'px');
+      assert.equal(pdfContainer().style.width, 'calc(100% - 428px)');
```

Comment:
Mind: lightly blown. I didn't realize you could do this particular `calc` operation. Wildly useful. I write this comment with a modicum of _I should have known this already_ shame :).